.github/workflows/installMediaWiki.sh

```diff
@@ -43,3 +43,6 @@ cat <<EOT >> composer.local.json
 	}
 }
 EOT
+
+cd extensions
+git clone https://gerrit.wikimedia.org/r/mediawiki/extensions/Wikibase --branch=$MW_BRANCH --recurse-submodules -j8
```

Comment:
This seems odd and is presumably overriding an already pulled, but wrong, version of Wikibase. Can't the original pulled version be made correct?

Reply:
Oddly enough, the original version gave
```
Run cd extensions/Wikibase ; git submodule update --init --recursive
fatal: not a git repository (or any of the parent directories): .git
Error: Process completed with exit code 128.
```
But if you want the original version to work, I can definitely investigate why that happens and try to get that version to work.

Reply:
Ah I see now you removed the original version